unpub/lib/src/app.dart

```diff
@@ -18,6 +18,7 @@ import 'package:unpub/src/package_store.dart';
 import 'utils.dart';
 import 'static/index.html.dart' as index_html;
 import 'static/main.dart.js.dart' as main_dart_js;
+import 'static/style.dart.css.dart' as style_dart_css;
```

Comment:
i don't find the file `style.dart.css.dart` or am i missing something?

Reply:
here “unpub/lib/src/static/main.dart.js.dart”

Reply:
I have added "style.dart.css.dart" file, please update and check~